frontend/views/containers/sidebar/GroupsList.vue

```diff
@@ -1,60 +1,33 @@
 <template lang='pug'>
-div(v-if='groupsByName.length')
-  menu-parent(@select='handleMenuSelect')
-    menu-trigger.c-trigger(:class="`has-text-${isDarkTheme ? 'white' : 'dark'}`")
-      avatar.c-avatar(
-        src='/assets/images/default-avatar.png'
-        :alt='groupSettings.groupName'
-        :blobURL='groupSettings.groupPicture'
-      )
-      | {{ groupSettings.groupName }}
-      i.icon-angle-down
+ul.c-group-list(v-if='groupsByName.length')
+  li.c-group-list-item.group-badge(
+    v-for='(group, index) in groupsByName'
+    :key='`group-${index}`'
+    tag='button'
+    :class="{ 'is-active': currentGroupId === group.contractID}"
+  )
+    avatar.c-avatar(
+      src='/assets/images/default-avatar.png'
+      :alt='group.groupName'
+      :blobURL='group.groupPicture'
+      @click="handleMenuSelect(group.contractID)"
```

Comment:
Each group is a clickable element, so it should be a `button` or an anchor if we redirect to another page. Since at the moment we keep the same URL, I believe a `button` is the best.

Reply:
Also, this doesn't seem to work properly. When I click, I need to refresh the page so it really changes the group dashboard, otherwise, the group scope is still the previous.

Reply:
The group is changing correctly in the vuex store. It also update the group in the selector component. There must be another problem coming from other components listeners.